### PR TITLE
Enable DNS certificate provisioning

### PR DESCRIPTION
**Pull request by lei-tang**
This PR is for issue: istio/istio#17061

Design documents:
[1]https://docs.google.com/document/d/1VHyBre8943USOx_YEVtA18KfEoGTmLT1iBHESO5bzcA/edit#heading=h.qex63c29z2to
[2] https://docs.google.com/document/d/1v8BxI07u-mby5f5rCruwF7odSXgb9G8-C9W5hQtSIAg/edit#heading=h.xw1gqgyqs5b

### CODE REVIEW

File: global.yaml [hunk 1]
```diff
@@ -464,6 +464,15 @@ global:
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
 
+  # Configures DNS certificates provisioned through Chiron linked into Pilot.
+  # Te values in this file are all hard-coded; please ensure the namespaces
+  # in dnsNames are consistent with those of your services.
+  certificates:
+    - secretName: dns.istio-galley-service-account
+      dnsNames: [istio-galley.istio-system.svc, istio-galley.istio-system]
```
Comment (rlenglet) [inline global.yaml]: Any way to parameterize these DNS names with the `istioNamespace` setting?

File: global.yaml [hunk 2]
```diff
@@ -464,6 +464,15 @@ global:
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
 
+  # Configures DNS certificates provisioned through Chiron linked into Pilot.
+  # Te values in this file are all hard-coded; please ensure the namespaces
```
Comment (rlenglet) [inline global.yaml]:
```suggestion
  # The DNS names are all hard-coded; please ensure the namespaces
```
Point to which values need to be kept in sync.
And in the comments for those values, add comments to point back to here.

File: global.yaml (hunk 1 above)
Comment (lei-tang) [reply global.yaml]: I have the same question; all the values in this file are hard-coded.
Add @costinm and @howardjohn regarding to this question.

File: global.yaml (hunk 2 above)
Comment (lei-tang) [reply global.yaml]: Done. For the config in istio-control/istio-discovery/templates/configmap.yaml that uses this field, the config already points back to here as: .Values.global.certificates.

File: global.yaml (hunk 1 above)
Comment (howardjohn) [reply global.yaml]: Not that I know of unfortunately

**Review by howardjohn — APPROVED** [review]

**Pull request merged**